LabExT/Movement/MoverNew.py

```diff
@@ -442,6 +445,39 @@ def _lift_lower_stages(lift):
         finally:
             _lift_lower_stages(-self.z_lift)
 
+    @assert_connected_stages
+    def move_to_device(self, chip: Type[Chip], device_id: Any):
```

Comment:
why not make the prototype directly: `move_to_device(self, device: Type[Device])`?

Reply:
Good point, I find the interface suboptimal as well.
The device class unfortunately has no reference to the chip (so can't do `device.chip` or something else) and I didn't want to change the device class as it is being customized by Nino at the moment. Would suggest that we adjust that when the new device implementation is in.